.github/scripts/main/hygiene.sh

```diff
@@ -124,4 +124,17 @@ please run dune exec --root=. -- ./ci.exe from .github/workflows and fixup the c
 fi
 (set +x ; echo -en "::endgroup::check workflow generation\r") 2>/dev/null
 
+###
+# Shellcheck
+###
+
+(set +x ; echo -en "::group::check shell scripts using shellcheck\r") 2>/dev/null
+if shellcheck shell/install.sh ; then
+  (set +x; echo "shell/install.sh: OK") 2>/dev/null
+else
+  (set +x; echo "shell/install.sh: ERROR") 2>/dev/null
```

Comment:
```suggestion
  (set +x; echo -e "shell/install.sh: [\e[31mERROR\e[0m]") 2>/dev/null
```

Reply:
done (although i didn't keep the brackets as they don't really make sense in this position)